src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(Color color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (int) (firstLeg * secondLeg * 0.5);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle"
+                + ", area: "
+                + calculateArea() + " sq.units, firstLeg: "
+                + firstLeg + " units, secondLeg: "
+                + secondLeg + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
[you need to use method `name()` to get a string representation of the enum. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)